src/applications/edu-benefits/feedback-tool/config/form.js

```diff
@@ -100,7 +100,7 @@ const formConfig = {
   trackingPrefix: 'gi_bill_feedback',
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
-  formId: 'complaint-tool',
+  formId: 'COMPLAINT-TOOL', // TODO: rename to feedback-tool once renamed on BE
```

Comment:
We will work with @lihanli and @jholton to coordinate updating this with the BE changes in another PR.

Reply:
Making this uppercase resolves the prefill issue for now.